src/Storage/NoTTLRateLimitStorageInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Bedrock\Bundle\RateLimitBundle\Storage;
+
+use Bedrock\Bundle\RateLimitBundle\Model\RateLimit;
+
+interface NoTTLRateLimitStorageInterface extends RateLimitStorageInterface
```

Comment:
I don't understand the name of this interface, what does it mean?

Reply:
`ResetableRateLimitStorageInterface` is a better name ?